README.md

```diff
@@ -754,6 +773,23 @@ in a column named `animal_name`.
 #### Constraints and Rules
 - Must be on a property field of list type.
 
+### not_contains
+#### Example Use
+```graphql
+{
+    Animal {
+        alias @filter(op_name: "not_contains", value: ["$wanted"])
+        name @output(out_name: "animal_name")
+    }
+}
+```
+This returns one row for every `Animal` whose list of aliases does not contain the value supplied
```

Comment:
Could make it clear here and in the surrounding parts of the README.md that you are referring to an instance of the class? You could maybe do this by adding `Vertex` after `Animal`.